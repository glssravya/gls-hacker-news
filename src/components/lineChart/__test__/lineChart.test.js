import React from 'react'
import {render, cleanup} from '@testing-library/react'
import LineChart from './../lineChart'

 afterEach(cleanup)
 
 it('should take a snapshot', () => {
    let chartData = new Map();
    chartData.set(1, 100);
    chartData.set(2, 200);

    const { asFragment } = render(<LineChart data={chartData}/>)
    
    expect(asFragment(<LineChart />)).toMatchSnapshot()
   });