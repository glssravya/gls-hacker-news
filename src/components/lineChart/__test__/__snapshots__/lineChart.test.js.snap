// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="lineChart"
  >
    <div
      data-highcharts-chart="0"
      style="overflow: hidden;"
    >
      <div
        class="highcharts-container "
        dir="ltr"
        id="highcharts-kniyv4w-0"
        style="position: relative; overflow: hidden; width: 600px; height: 400px; text-align: left; line-height: normal; z-index: 0;"
      >
        <svg
          class="highcharts-root"
          height="400"
          style="font-family:\\"Lucida Grande\\", \\"Lucida Sans Unicode\\", Arial, Helvetica, sans-serif;font-size:12px;"
          version="1.1"
          viewBox="0 0 600 400"
          width="600"
          xmlns="http://www.w3.org/2000/svg"
        >
          <desc>
            Created with Highcharts 8.0.4
          </desc>
          <defs>
            <clippath
              id="highcharts-kniyv4w-1-"
            >
              <rect
                fill="none"
                height="NaN"
                width="NaN"
                x="0"
                y="0"
              />
            </clippath>
            <clippath
              id="highcharts-kniyv4w-4-"
            >
              <rect
                fill="none"
                height="400"
                width="NaN"
                x="-99"
                y="-10"
              />
            </clippath>
            <clippath
              id="highcharts-kniyv4w-5-"
            >
              <rect
                fill="none"
                height="NaN"
                width="NaN"
                x="0"
                y="0"
              />
            </clippath>
          </defs>
          <rect
            class="highcharts-background"
            fill="rgb(246, 246, 239)"
            height="400"
            rx="0"
            ry="0"
            width="600"
            x="0"
            y="0"
          />
          <rect
            class="highcharts-plot-background"
            fill="none"
            height="NaN"
            width="NaN"
            x="NaN"
            y="10"
          />
          <g
            class="highcharts-grid highcharts-xaxis-grid"
            data-z-index="1"
          >
            <path
              class="highcharts-grid-line"
              d="M 0 0"
              data-z-index="1"
              fill="none"
              opacity="1"
            />
            <path
              class="highcharts-grid-line"
              d="M 0 0"
              data-z-index="1"
              fill="none"
              opacity="1"
            />
            <path
              class="highcharts-grid-line"
              d="M 0 0"
              data-z-index="1"
              fill="none"
              opacity="1"
            />
          </g>
          <g
            class="highcharts-grid highcharts-yaxis-grid"
            data-z-index="1"
          >
            <path
              class="highcharts-grid-line"
              d="M 0 0"
              data-z-index="1"
              fill="none"
              opacity="1"
              stroke="#e6e6e6"
              stroke-width="1"
            />
            <path
              class="highcharts-grid-line"
              d="M 0 0"
              data-z-index="1"
              fill="none"
              opacity="1"
              stroke="#e6e6e6"
              stroke-width="1"
            />
            <path
              class="highcharts-grid-line"
              d="M 0 0"
              data-z-index="1"
              fill="none"
              opacity="1"
              stroke="#e6e6e6"
              stroke-width="1"
            />
            <path
              class="highcharts-grid-line"
              d="M 0 0"
              data-z-index="1"
              fill="none"
              opacity="1"
              stroke="#e6e6e6"
              stroke-width="1"
            />
            <path
              class="highcharts-grid-line"
              d="M 0 0"
              data-z-index="1"
              fill="none"
              opacity="1"
              stroke="#e6e6e6"
              stroke-width="1"
            />
            <path
              class="highcharts-grid-line"
              d="M 0 0"
              data-z-index="1"
              fill="none"
              opacity="1"
              stroke="#e6e6e6"
              stroke-width="1"
            />
            <path
              class="highcharts-grid-line"
              d="M 0 0"
              data-z-index="1"
              fill="none"
              opacity="1"
              stroke="#e6e6e6"
              stroke-width="1"
            />
            <path
              class="highcharts-grid-line"
              d="M 0 0"
              data-z-index="1"
              fill="none"
              opacity="1"
              stroke="#e6e6e6"
              stroke-width="1"
            />
          </g>
          <rect
            class="highcharts-plot-border"
            data-z-index="1"
            fill="none"
            height="0"
            width="0"
            x="0"
            y="10"
          />
          <g
            class="highcharts-axis highcharts-xaxis"
            data-z-index="2"
          >
            <text
              class="highcharts-axis-title"
              data-z-index="7"
              style="color:0,0,0;font-weight:bold;font-size:18px;fill:0,0,0;"
              text-anchor="middle"
              transform="translate(0,0)"
              x="0"
              y="-9999"
            >
              <tspan
                style="font-weight:bold"
              >
                ID
              </tspan>
            </text>
            <path
              class="highcharts-axis-line"
              d="M 0 0"
              data-z-index="7"
              fill="none"
              stroke="#ccd6eb"
              stroke-width="1"
            />
          </g>
          <g
            class="highcharts-axis highcharts-yaxis"
            data-z-index="2"
          >
            <text
              class="highcharts-axis-title"
              data-z-index="7"
              style="color:0,0,0;font-weight:bold;font-size:18px;fill:0,0,0;"
              text-anchor="middle"
              transform="translate(0,0) rotate(270 0 -9999)"
              x="0"
              y="-9999"
            >
              <tspan
                style="font-weight:bold"
              >
                Votes
              </tspan>
            </text>
            <path
              class="highcharts-axis-line"
              d="M 0 0"
              data-z-index="7"
              fill="none"
              stroke="#ccd6eb"
              stroke-width="1"
            />
          </g>
          <g
            class="highcharts-series-group"
            data-z-index="3"
          >
            <g
              class="highcharts-series highcharts-series-0 highcharts-line-series"
              clip-path="url(#highcharts-kniyv4w-5-)"
              data-z-index="0.1"
              transform="translate(0,10) scale(1 1)"
            >
              <path
                class="highcharts-graph"
                d="M -100000 -100000 L -100000 -100000"
                data-z-index="1"
                fill="none"
                stroke="steelblue"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="2"
              />
              <path
                class="highcharts-tracker-line"
                d="M -100010 -100000 L -100000 -100000 L -100000 -100000 L -99990 -100000"
                data-z-index="2"
                fill="none"
                stroke="rgba(192,192,192,0.002)"
                stroke-linejoin="round"
                stroke-width="22"
                visibility="visible"
              />
            </g>
            <g
              class="highcharts-markers highcharts-series-0 highcharts-line-series highcharts-tracker"
              clip-path="url(#highcharts-kniyv4w-4-)"
              data-z-index="0.1"
              transform="translate(0,10) scale(1 1)"
            />
          </g>
          <text
            class="highcharts-title"
            data-z-index="4"
            style="color:#333333;font-size:18px;fill:#333333;"
            text-anchor="middle"
            x="300"
            y="24"
          />
          <text
            class="highcharts-subtitle"
            data-z-index="4"
            style="color:#666666;fill:#666666;"
            text-anchor="middle"
            x="300"
            y="10"
          />
          <text
            class="highcharts-caption"
            data-z-index="4"
            style="color:#666666;fill:#666666;"
            text-anchor="start"
            x="10"
            y="397"
          />
          <g
            class="highcharts-legend"
            data-z-index="7"
          >
            <rect
              class="highcharts-legend-box"
              fill="none"
              height="8"
              rx="0"
              ry="0"
              visibility="hidden"
              width="8"
              x="0"
              y="0"
            />
            <g
              data-z-index="1"
            >
              <g />
            </g>
          </g>
          <g
            class="highcharts-axis-labels highcharts-xaxis-labels"
            data-z-index="7"
          >
            <text
              style="color:#666666;cursor:default;font-size:11px;fill:#666666;"
              text-anchor="middle"
              transform="translate(0,0)"
              x="0"
            >
              1
            </text>
            <text
              style="color:#666666;cursor:default;font-size:11px;fill:#666666;"
              text-anchor="middle"
              transform="translate(0,0)"
              x="0"
            >
              2
            </text>
          </g>
          <g
            class="highcharts-axis-labels highcharts-yaxis-labels"
            data-z-index="7"
          >
            <text
              style="color:#666666;cursor:default;font-size:11px;fill:#666666;"
              text-anchor="end"
              transform="translate(0,0)"
              x="0"
            >
              80
            </text>
            <text
              style="color:#666666;cursor:default;font-size:11px;fill:#666666;"
              text-anchor="end"
              transform="translate(0,0)"
              x="0"
            >
              100
            </text>
            <text
              style="color:#666666;cursor:default;font-size:11px;fill:#666666;"
              text-anchor="end"
              transform="translate(0,0)"
              x="0"
            >
              120
            </text>
            <text
              style="color:#666666;cursor:default;font-size:11px;fill:#666666;"
              text-anchor="end"
              transform="translate(0,0)"
              x="0"
            >
              140
            </text>
            <text
              style="color:#666666;cursor:default;font-size:11px;fill:#666666;"
              text-anchor="end"
              transform="translate(0,0)"
              x="0"
            >
              160
            </text>
            <text
              style="color:#666666;cursor:default;font-size:11px;fill:#666666;"
              text-anchor="end"
              transform="translate(0,0)"
              x="0"
            >
              180
            </text>
            <text
              style="color:#666666;cursor:default;font-size:11px;fill:#666666;"
              text-anchor="end"
              transform="translate(0,0)"
              x="0"
            >
              200
            </text>
            <text
              style="color:#666666;cursor:default;font-size:11px;fill:#666666;"
              text-anchor="end"
              transform="translate(0,0)"
              x="0"
            >
              220
            </text>
          </g>
        </svg>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
